synapse/storage/databases/main/monthly_active_users.py

```diff
@@ -210,10 +211,10 @@ def _reap_users(txn, reserved_users):
             # is racy.
             # Have resolved to invalidate the whole cache for now and do
             # something about it if and when the perf becomes significant
-            self._invalidate_all_cache_and_stream(
+            self._invalidate_all_cache_and_stream(  # type: ignore[attr-defined]
```

Comment:
Urgh. Fixing this properly would require us to inherit from `CacheInvalidationWorkerStore`. [I've been down that road](https://github.com/matrix-org/synapse/pull/11354), and it's not pretty.